string receiverUUID
string publisherUUID
string publisherFeatureUUID
string receiverFeatureUUID
float64 coefficient
